Winapp2.ini

```diff
@@ -441,105 +325,131 @@ FileKey5=%ProgramFiles%\Google\Chrome\Application\*\Installer|*.7z
 [JumpListIconsRecentClosed *]
```

Comment:
I think this entry should perhaps be merged into History *

Reply:
Addtly, it looks like there's another path here called "JumpListIconsMostVisited" that can be similarly cleaned. Both are for the jumplist that opens when you right click chrome on the windows task bar.